src/cascadia/TerminalApp/AppKeyBindings.idl

```diff
@@ -9,6 +9,7 @@ namespace TerminalApp
         CopyTextWithoutNewlines,
         PasteText,
         NewTab,
+		DuplicateTab,

```

Comment:
I think you have tabs configured instead of spaces. This doesn't appear to be indented correctly.